docs/design/2020-06-01-global-kill.md

```diff
@@ -17,28 +17,34 @@ Currently connection ids are local to TiDB instances, which means that a `KILL x
 To support "Global Kill", we need:
 1. Global connection ids, which are unique among all TiDB instances.
 2. Redirect `KILL x` to target TiDB instance, on which the connection `x` is running.
-3. Support both 32 & 64 bits `connId`, to be compatible with legacy 32 bits clients. In this stage, we only design the 64 bits `connId`, and left a `markup` to distinguish between these two kinds.
+3. Support both 32 & 64 bits `connId`. 32 bits `connId` is used on small clusters (number of TiDB instances less than 2048), to be fully compatible with all clients including legacy 32 bits ones, while 64 bits `connId` is used for big clusters. Bit 0 in `connId` is a markup to distinguish between these two kinds.
 
 ## Rationale
 
 #### 1. Structure of `connId`
-##### 64 bits version
+##### 32 bits
+```
+                                      31    21 20               1    0
+                                     +--------+------------------+------+
+                                     |serverId|   local connId   |markup|
```

Comment:
```suggestion
                                     |serverID|   local connID   |markup|
```

There are similar problems in other places, which need to be corrected together

Reply:
Done. PTAL, thx~ @zimulala